cfgov/v1/preprocessed/css/molecules/image-and-text-25-75.less

```diff
@@ -0,0 +1,93 @@
+/* topdoc
+  name: Image and Text 25/75
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-image-and-text-25-75">
+            <div class="m-image-and-text-25-75_image-container">
+                <img src="image">
+            </div>
+            <div class="m-image-and-text-25-75_content">
+                <h1 class="m-image-and-text-25-75_heading">Heading</h1>
+                <p class="m-image-and-text-25-75_body">Body content</p>
+                <a class="m-image-and-text-25-75_link">Link</a>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-image-and-text-25-75
+            .m-image-and-text-25-75_image-container
+              img
+            .m-image-and-text-25-75_content
+              .m-image-and-text-25-75_heading
+              p.m-image-and-text-25-75_body
+              a.m-image-and-text-25-75_link
+  tags:
+    - cfgov-molecules
+*/
+
+.m-image-and-text-25-75 {
+
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+
+    &_image-container {
+
+        > img {
```

Comment:
In the GHE issue we discussed that there may be situations where this will be a bg image in css. If that's still true there should just be a classname like `&_image` that's used without nesting in the container.